google-cloud-firestore/src/main/java/com/google/cloud/firestore/encoding/CustomClassMapper.java

```diff
@@ -38,7 +43,7 @@
 import java.util.concurrent.ConcurrentMap;
 
 /** Helper class to convert to/from custom POJO classes and plain Java types. */
-class CustomClassMapper {
+public class CustomClassMapper {
```

Comment:
Is it a good idea to make this class and the methods below public, exposing them to library users?

Reply:
I am also hesitating over this.

Reply:
added "@InternalApi" to avoid public use.